.gitignore

```diff
@@ -33,3 +33,6 @@ Icon
 Network Trash Folder
 Temporary Items
 .apdisk
+
+# Microsoft Visual Studio 
+.vscode/
```

Comment:
not here 😉 
Add it to your global .gitignore pls

Reply:
I usually set this on a per-project (as for other projects, I'm actually versioning the VSCode stuffs). Hence I did the same here :) I removed it from this file.